apps/dashboard/src/components/workflow-editor/steps/digest/utils.ts

```diff
@@ -0,0 +1,403 @@
+import cronParser, {
```

Comment:
These are the utils used by the scheduled digest UI to calculate the period from the `cron` and based on the current period value selected (when switching). 
There are also utils that convert the cron fields to UI field values because the `cron-parser` for `*` generates the array with all values, but on the UI, we use an empty array as a sign for `*` basically `every` day, month, hour etc.